src/main/scala/dpla/ingestion3/mappers/providers/EsdnMapping.scala

```diff
@@ -0,0 +1,220 @@
+package dpla.ingestion3.mappers.providers
+
+import java.net.URI
+
+import dpla.ingestion3.enrichments.normalizations.StringNormalizationUtils._
+import dpla.ingestion3.enrichments.normalizations.filters.{DigitalSurrogateBlockList, FormatTypeValuesBlockList}
+import dpla.ingestion3.mappers.utils.{Document, IdMinter, Mapping, XmlExtractor}
+import dpla.ingestion3.messages.{IngestMessage, IngestMessageTemplates, MessageCollector}
+import dpla.ingestion3.model.DplaMapData._
+import dpla.ingestion3.model.{nameOnlyAgent, _}
+import dpla.ingestion3.utils.Utils
+import org.json4s.JValue
+import org.json4s.JsonDSL._
+
+import scala.util.{Failure, Success, Try}
+import scala.xml._
+
+
+class EsdnMapping extends Mapping[NodeSeq] with XmlExtractor with IdMinter[NodeSeq] with IngestMessageTemplates {
+
+  val formatBlockList: Set[String] =
+    DigitalSurrogateBlockList.termList ++
+      FormatTypeValuesBlockList.termList
+
+  // ID minting functions
+  override def useProviderName(): Boolean = true
+
+  override def getProviderName(): String = "esdn"
+
+  override def getProviderId(implicit data: Document[NodeSeq]): String = // TODO confirm w/gretchen
```

Comment:
Sanity check - did you want to confirm w/ gretchen before or after this is merged?